src/test/java/io/confluent/idesidecar/restapi/auth/RefreshCCloudTokensBeanTest.java

```diff
@@ -27,7 +28,7 @@ class RefreshCCloudTokensBeanTest {
   @Inject
   RefreshCCloudTokensBean refreshCCloudTokensBean;
 
-  @Test
+  @RetryingTest(maxAttempts = 5)
```

Comment:
I added the `@RetryingTest` annotation because I noticed that this test fails randomly from time to time. We should try to figure out the underlying issue once time permits.